### PR TITLE
Remove randomness from percy snapshots

### PR DESCRIPTION
**Pull request by callumacrae**
### Context

it was annoying and bad

### Guidance to review

percy will error in this PR, but once merged to develop should start working!

### How can I view this in a review app?

you can't

### CODE REVIEW

File: spec/cypress/app_commands/scenarios/admin/schools.rb [hunk 1]
```diff
@@ -3,7 +3,9 @@
 school = FactoryBot.create(:school, name: "Include this school", urn: 123_456)
 local_authority = FactoryBot.create(:local_authority)
 SchoolLocalAuthority.create!(school: school, local_authority: local_authority, start_year: 2021)
-FactoryBot.create(:user, :induction_coordinator, full_name: "Sarah Smith", email: "sarah.smith@example.com", schools: [school])
```
Comment (callumacrae) [inline spec/cypress/app_commands/scenarios/admin/schools.rb:6]: This was an interesting one - creating a user with an induction_coordinator profile would create a school which had a constant name but a random URN.

**Review by kicferk1 — APPROVED** [review]
I would ask "is that all?", but I guess there is no easy way of finding out until we merge this, right?

**Comment by callumacrae** [comment]
> I would ask "is that all?", but I guess there is no easy way of finding out until we merge this, right?

See percy on this PR - there's only two snapshots containing randomness! So yes, this should be all :)

File: spec/cypress/integration/admin/SchoolManagment.feature [hunk 2]
```diff
@@ -26,8 +26,9 @@ Feature: Admin user managing schools
     And the page should be accessible
     And percy should be sent snapshot
 
+  @focus
   Scenario: Viewing a list of schools
-    Then the table should have 20 rows
+    Then the table should have 12 rows
```
Comment (callumacrae) [inline spec/cypress/integration/admin/SchoolManagment.feature:31]: This isn't directly related to this PR, but I was concerned that it was testing for 20 rows when there were actually 23 schools and it was just testing the pagination.

**Pull request merged**